package.bzl

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Package file which defines build_bazel_rules_nodejs version in skylark
+
+check_rules_nodejs_version can be used in downstream WORKSPACES to check
+against a minimum dependent build_bazel_rules_nodejs version.
+"""
+
+load("//internal/common:parse_version.bzl", "parse_version")
+
+VERSION = "0.11.2"
+
+def check_rules_nodejs_version(minimum_version_string):
+  """
+  Verify that a miminum build_bazel_rules_nodejs is loaded a WORKSPACE.
```

Comment:
spelling: minimum

Reply:
thanks 🙄  I need a spell check in my vscode.